app/tests/spec/models/reliers/oauth.js

```diff
@@ -252,8 +252,8 @@ define([
           scope: SCOPE
         });
 
-        oAuthClient.getClientInfo.restore();
-        sinon.stub(oAuthClient, 'getClientInfo', function () {
+        fxaClient.getOAuthClientInfo.restore();
+        sinon.stub(fxaClient, 'getOAuthClientInfo', function () {
           return p.reject(OAuthErrors.toError('INVALID_REQUEST_SIGNATURE'));
```

Comment:
See the [question above](https://github.com/mozilla/fxa-content-server/pull/2640/files#r33589652) about how errors will be handled.